icu4c/source/tools/genrb/parse.cpp

```diff
@@ -2000,6 +2000,10 @@ parse(UCHARBUF *buf, const char *inputDir, const char *outputDir, const char *fi
 
     if (state.bundle == NULL || U_FAILURE(*status))
     {
+        if (state.bundle) {
```

Comment:
The 'if' is not needed, just the delete.

Reply:
Done, thanks!